src/components/Search/Search.css

```diff
@@ -0,0 +1,34 @@
+.search-field {
```

Comment:
Love that you already jumped into that styling. Joana style 💞

Reply:
Yes, my favorite searching type hehe